src/data/models/institure_holidays.js

```diff
@@ -4,15 +4,17 @@ import DataTypes from 'sequelize';
 import Model from '../sequelize';
 
 const Institure_Holidays = Model.define('institure_holidays', {
-  id: {
-    type: DataTypes.INTEGER(11).UNSIGNED,
-    allowNull: false,
-    autoIncrement: true,
-    primaryKey: true,
-  },
+  // required?
+  // id: {
+  //   type: DataTypes.INTEGER(11).UNSIGNED,
+  //   allowNull: false,
+  //   autoIncrement: true,
+  //   primaryKey: true,
+  // },
   date: {
```

Comment:
Might as well change it to [DATE](http://docs.sequelizejs.com/variable/index.html#static-variable-DataTypes) DataType.